_pages/es/index.md

```diff
@@ -27,7 +27,7 @@ Defend Internet Privacy and Policy (coming soon)
 
 {: #area-testpilot .area}
 [![image](/asserts/img/test-pilot.png)
-Use and test experimental Firefox features](/experiments/)
+Use and test experimental Firefox features](/es/experiments/)
```

Comment:
No, Jekyll didn't handle it, you could test it in the current website, if you chose es, links still go to english version